kinesis/src/main/scala/akka/stream/alpakka/kinesis/impl/SetupStage.scala

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.kinesis.impl
+
+import akka.annotation.InternalApi
+import akka.stream._
+import akka.stream.scaladsl.{Flow, Keep, Sink, Source}
+import akka.stream.stage._
+
+import scala.concurrent.{Future, Promise}
+
+@InternalApi
+private[kinesis] final class SetupSinkStage[T, M](factory: ActorMaterializer => Attributes => Sink[T, M])
```

Comment:
The setup-stages are now available as standard operators in Akka. Please change to use those.